Entities/ACraft.cpp

```diff
@@ -983,22 +994,21 @@ void ACraft::Update()
 // Description:     Draws this Actor's current graphical HUD overlay representation to a
 //                  BITMAP of choice.
 
-void ACraft::DrawHUD(BITMAP *pTargetBitmap, const Vector &targetPos, int whichScreen, bool playerControlled)
-{
+void ACraft::DrawHUD(BITMAP *pTargetBitmap, const Vector &targetPos, int whichScreen, bool playerControlled) {
+	m_HUDStack = -m_CharHeight / 2;
```

Comment:
Why did you feel the need to do this when it's already done in Actor? Just for future proofing?

Reply:
Because the function returns at `!HUDVisible` before it gets to `Actor::DrawHUD`

Reply:
Ahhh, and it's probably helpful for lua to have a correct hudstack?

Reply:
HUDStack isn't directly available to lua but it affects `AboveHUDPos` which is often used to place objective markers and other stuff which sometimes happens to actors with invisible HUD, see the crabs in "cave exploration" for example (unless I already pushed changes that use radius instead)

Reply:
Yeah, having AboveHUDPos work properly is definitely a good thing. Okay, I'm fine with this, feel free to resolve.